BUILD.bazel

```diff
@@ -16,6 +17,14 @@ exports_files([
     "package.json",
 ])
 
+# Files required by e2e tests
+copy_to_bin(
+    name = "dot-files",
```

Comment:
This is not a dot file right?  Were others included at one point?

Reply:
I assume it was at one point and has evolved over time. I'll change it 👍